Sources/carton-release/HashArchive.swift

```diff
@@ -52,7 +52,11 @@ struct HashArchive: ParsableCommand {
       localFileSystem.readFileContents(AbsolutePath(staticPath, "bundle.js"))
     ).hexadecimalRepresentation.uppercased()
 
-    try ProcessRunner(["zip", "static.zip", "static/*"], terminal).waitUntilFinished()
+    let archiveSources = try localFileSystem.traverseRecursively(staticPath)
+      .dropFirst()
```

Comment:
Please add comment to indicate that `dropFirst` drops root directory path returned by `traverseRecursively`